### PR TITLE
cp: add French translation

### PR DESCRIPTION
**Pull request by neel24**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/common/cp.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`cp -vR {{chemin/vers/répertoire_source}} {{chemin/vers/répertoire_cible}}`
+
+- Copiez les fichiers texte vers un autre emplacement, en mode interactif (invite l'utilisateur avant d'écraser):
+
+`cp -i {{*.txt}} {{chemin/vers/répertoire_cible}}`
+
+- Déréférencer les liens symboliques avant de copier:
+
+`cp -L {{link}} {{chemin/vers/répertoire_cible}}`
```
Comment (navarroaxel) [inline pages.fr/common/cp.md:27]: you forgot the `--parents` example. Same happened is [this translation PR](https://github.com/tldr-pages/tldr/pull/4571) lol

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @neel24 thanks for your submission. I left a comment.

File: pages.fr/common/cp.md (hunk 1 above)
Comment (neel24) [reply pages.fr/common/cp.md:27]: My bad, I had only looked at the cp.md file in pages/common :)

File: pages.fr/common/cp.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# cp
+
+> Copiez des fichiers et des répertoires.
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:3]:
```suggestion
> Copie des fichiers et des répertoires.
```

File: pages.fr/common/cp.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- Copiez un fichier dans un autre emplacement:
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:5]:
```suggestion
- Copier un fichier vers un autre emplacement :
```

File: pages.fr/common/cp.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+`cp {{chemin/vers/fichier_source.ext}} {{chemin/vers/fichier_cible.ext}}`
+
+- Copiez un fichier dans un autre répertoire, en conservant le nom du fichier:
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:9]:
```suggestion
- Copier un fichier vers un autre répertoire en conservant le nom du fichier :
```

File: pages.fr/common/cp.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+`cp {{chemin/vers/fichier_source.ext}} {{chemin/vers/répertoire_parent_cible}}`
+
+- Copiez récursivement le contenu d'un répertoire vers un autre emplacement (si la destination existe, le répertoire est copié à l'intérieur):
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:13]:
```suggestion
- Copier récursivement le contenu d'un répertoire vers un autre emplacement (si la destination existe, le répertoire est copié à l'intérieur) :
```

File: pages.fr/common/cp.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+`cp -R {{chemin/vers/répertoire_source}} {{chemin/vers/répertoire_cible}}`
+
+- Copiez un répertoire récursivement, en mode verbeux (affiche les fichiers au fur et à mesure de leur copie):
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:17]:
```suggestion
- Copier un répertoire récursivement, en mode verbeux (affiche les fichiers au fur et à mesure de leur copie) :
```

File: pages.fr/common/cp.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+`cp -vR {{chemin/vers/répertoire_source}} {{chemin/vers/répertoire_cible}}`
+
+- Copiez les fichiers texte vers un autre emplacement, en mode interactif (invite l'utilisateur avant d'écraser):
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:21]:
```suggestion
- Copier les fichiers texte vers un autre emplacement, en mode interactif (demande confirmation avant d'écraser) :
```

File: pages.fr/common/cp.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+`cp -i {{*.txt}} {{chemin/vers/répertoire_cible}}`
+
+- Déréférencer les liens symboliques avant de copier:
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:25]:
```suggestion
- Déréférencer les liens symboliques avant de copier :
```

File: pages.fr/common/cp.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+`cp -L {{link}} {{chemin/vers/répertoire_cible}}`
+
+- Utilisez le chemin complet des fichiers source, en créant les répertoires intermédiaires manquants lors de la copie:
```
Comment (Deuchnord) [inline pages.fr/common/cp.md:29]:
```suggestion
- Copier en utilisant le chemin complet des fichiers source, en créant les répertoires intermédiaires manquants :
```

**Review by Deuchnord — CHANGES_REQUESTED** [review]
Hey, nice translation!
I suggested some improvements :)

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @neel24 I didn't notice that the space before the `:` is missing.
and what do you think about the comments from @Deuchnord ?

File: pages.fr/common/cp.md (hunk 9 above)
Comment (neel24) [reply pages.fr/common/cp.md:29]: Hey @Deuchnord @navarroaxel, I agree with these changes as they definitely simplify the sentences and I had actually forgotten that in french a space is used before semicolons lol. I've made the requested changes with the commit below.

**Review by navarroaxel — APPROVED** [review]

**Comment by neel24** [comment]
> Hey @neel24 I didn't notice that the space before the `:` is missing.
> and what do you think about the comments from @Deuchnord ?

Hey @Deuchnord @navarroaxel, I agree with these changes as they definitely simplify the sentences and I had actually forgotten that in french a space is used before semicolons lol

**Review by Deuchnord — APPROVED** [review]

**Comment by sbrl** [comment]
Hey there, @Deuchnord! Thanks for the approval! Unfortunately it doesn't count towards the 2 required reviewers, but if you submit 5 non-trivial PRs and have them merged, then in line with our [community roles](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md) document you'd be eligible to become a contributor, which gives you the ability to leave reviews and merge PRs :D

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

Special thanks for @Deuchnord for the review here :D

**Pull request merged**